backstory/src/main/java/com/google/sps/story/data/DatamuseRequestClient.java

```diff
@@ -55,61 +59,79 @@ public DatamuseRequestClient(String url) {
   }
 
   /**
-   * Fetches an array of adjectives related to the passed-in noun, which will have
-   * a passed-in cap on its size (array returned might be smaller if database
-   * does not have enough related adjectives). Fetches these adjectives
-   * by querying the Datamuse database.
+   * Returns a randomly-chosen topic related to storytelling
+   * from a preset array.
    *
-   * @param noun the noun to get adjectives related to (must be one word)
-   * @param cap the maximum number of adjectives to retrieve
-   * @return an array of adjectives related to the noun
+   * @return a topic related to storytelling to be used in the PCS
+   */
+  public static String getStorytellingTopic() {
```

Comment:
`getRandomStorytellingTopic()` would more clearly indicate what this method does, if the reader doesn't come read the documentation.

Reply:
That makes sense. I'll make the change.

Reply:
Done